doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_rest_handler.md

```diff
@@ -2,126 +2,112 @@
 Content:
 
 * [Functionality](#section1)
-    * [Mapping NGSI events to flume events](#section1.1)
-    * [Additional headers added by Flume interceptors](#section1.2)
-    * [Example](#section1.3)
+    * [Mapping NGSI events to `NGSIEvent` objects](#section1.1)
+    * [Example](#section1.2)
 * [Administration guide](#section2)
     * [Configuration](#section2.1)
     * [Accepted character set](#section2.2)
 * [Programmers guide](#section3)
     * [`NGSIRestHandler` class](#section3.1)
 
 ##<a name="section1"></a>Functionality
-###<a name="section1.1"></a>Mapping NGSI events to flume events
-This section explains how a notified NGSI event (an http message) containing context data is converted into a Flume event (an object in memory or a file), suitable for being consumed by any of the Cygnus sinks, thanks to `NGSIRestHandler`.
+###<a name="section1.1"></a>Mapping NGSI events to `NGSIEvent` objects
+This section explains how a notified NGSI event (a http message containing headers and payloa) is used to crate a `NGSIEvent` object, suitable for being consumed by any of the Cygnus sinks, thanks to `NGSIRestHandler`.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1308/commits/7dbe4ab1ef9d2cc343af43c0e0fe9c6a960dc4af